src/components/app.js

```diff
@@ -39,24 +34,32 @@ class App extends React.Component {
               <h3 className='qpp-docs-title-text'>QPP Submissions API Documentation</h3>
             </div>
           </div>
-          <div className='content-block'>
-            <div className='content-container ds-l-row'>
-              <div className='ds-u-float--left ds-u-padding-right--6 ds-u-padding-top--2 ds-l-col--3'>
-                <ul className='ds-c-vertical-nav__subnav'>
+
+          <div className='content-container ds-l-row'>
+            <div id='dropdown-nav' className='ds-l-col--12 ds-u-padding-top--3'>
+              <form>
+                <DropdownNav />
+              </form>
+            </div>
+
+            <div id='left-nav' className='ds-l-col--3 ds-u-padding-top--4'>
+              <div className='content-block'>
+                <ul className='ds-c-list--bare ds-u-padding-top--2'>
                   {LeftNav}
                 </ul>
               </div>
-              <div className='ds-u-float--left ds-u-padding--1 ds-l-col--9'>
-                <div className='ds-u-measure--wide'>
-                  {mergedRoutes.map(({path, exact, component}, index) => (
-                    <Route
-                      key={index}
-                      path={'/' + path}
-                      exact={exact}
-                      component={component}
-                      />
-                  ))}
-                </div>
+            </div>
+
+            <div className='ds-u-float--left ds-u-padding-top--4 ds-l-col--12 ds-l-sm-col--9'>
```

Comment:
I think you can get rid of the `ds-u-float--left` since this is a flexbox column